.hmy/pgp_keys.asc

```diff
@@ -0,0 +1,29 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Shall we move this file to other repository?

Reply:
this is the public key, better reside in the main repo for ppl to trust, IMO.